### PR TITLE
Update packages versions to latest stable

### PR DESCRIPTION
**Pull request by elfosardo**

### CODE REVIEW

**Comment by elfosardo** [comment]
/hold
syncing latest packages before respinning the jobs

**Comment by rhjanders** [comment]
From what I see, we are on to the next problem: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_ironic-image/245/pull-ci-openshift-ironic-image-master-prevalidation-images/1470841622450147328#1:build-log.txt%3A780

Now, python3-autopage is missing.

**Comment by elfosardo** [comment]
tagged the missing packages, need to wait for the sync before retesting

File: main-packages-list.ocp [hunk 1]
```diff
@@ -13,18 +13,19 @@ ipxe-roms-qemu
...
 mod_ssl
-openstack-ironic-api >= 1:18.3.0-0.20211206200515.0ff3da2.el8
-openstack-ironic-conductor >= 1:18.3.0-0.20211206200515.0ff3da2.el8
-openstack-ironic-inspector >= 10.7.1-0.20210722154052.edf655c.el8
+openstack-ironic-api >= 1:19.0.1-0.20211210165034.c86ca20.el8
+openstack-ironic-conductor >= 1:19.0.1-0.20211210165034.c86ca20.el8
+openstack-ironic-inspector >= 10.9.1-0.20211209161020.21209f3.el8
 parted
 psmisc
+python3-cliff >= 3.9.0-0.20211020191737.734bc0c.el8
```
Comment (dtantsur) [inline main-packages-list.ocp:21]: I know it's useful for testing, but we shouldn't have cliff in our packages list.

Comment (elfosardo) [reply main-packages-list.ocp:21]: good point, this was actually for testing purpose only, I have to remove it

**Comment by zaneb** [comment]
Local builds of this are working for me, so I'm not sure why CI isn't happy?
/test images
/test prevalidation-e2e-metal-ipi-prevalidation

**Comment by elfosardo** [comment]
> Local builds of this are working for me, so I'm not sure why CI isn't happy? /test images /test prevalidation-e2e-metal-ipi-prevalidation

@zaneb if the repository is not in sync, the build fails

**Review by lhh — APPROVED** [review]

**Comment by zaneb** [comment]
Looks like a temporary quay problem
/retest-required

**Pull request merged**